Reqnroll/Reqnroll.nuspec

```diff
@@ -39,12 +37,13 @@
   <files>
     <file src="build\**\*" target="build" />
 
-    <file src="bin\$config$\net462\Reqnroll.*" exclude="bin\$config$\net462\Reqnroll.BoDi.*" target="lib\$Reqnroll_FullFramework_Runtime_TFM$" />
+    <file src="bin\$config$\net462\Reqnroll.*" target="lib\$Reqnroll_FullFramework_Runtime_TFM$" />
     <file src="Reqnroll.ExternalAnnotations.xml" target="lib\$Reqnroll_FullFramework_Runtime_TFM$" />
 
     <file src="bin\$config$\netstandard2.0\Reqnroll.dll" target="lib\$Reqnroll_Core_Runtime_TFM$" />
     <file src="bin\$config$\netstandard2.0\Reqnroll.pdb" target="lib\$Reqnroll_Core_Runtime_TFM$" />
     <file src="bin\$config$\netstandard2.0\Reqnroll.xml" target="lib\$Reqnroll_Core_Runtime_TFM$" />
+    <file src="bin\$config$\netstandard2.0\Reqnroll.BoDi.*" target="lib\$Reqnroll_Core_Runtime_TFM$" />
```

Comment:
I was wondering why for `net462` we include `Reqnroll.*` but for `netstandard2.0` we choose to select the individual `dll`, `pdb` and `xml`. Wouldn't it make sense to use `Reqnroll.*` there as well? (or vice versa)

Reply:
@mbhoek The netstandard compilation also generates a `deps.json` file as well. With `*` that would be also included. I also don't like this, but I hope we can anyway merge this to a single target soon, that's why it did not touch it.